tests/test_ci_driver_contract.py

```
@@ -252,3 +252,27 @@ def test_ci_workflow_jobs_have_explicit_timeout_bounds() -> None:
     assert set(CI_JOB_TIMEOUTS) == set(jobs)
     for job_name, expected_timeout in CI_JOB_TIMEOUTS.items():
         assert jobs[job_name].get("timeout-minutes") == expected_timeout
+
+
+def test_ci_driver_passes_shell_syntax() -> None:
+    """The shared CI driver must be parseable by bash."""
+    result = subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)